rosdep/python.yaml

```
@@ -5899,6 +5899,13 @@ python3-empy:
   openembedded: [python3-empy@meta-ros-common]
   rhel: ['python%{python3_pkgversion}-empy']
   ubuntu: [python3-empy]
+python3-evdev:
```

Here's the equivalent Arch package: https://archlinux.org/packages/extra/x86_64/python-evdev/
```suggestion
python3-evdev:
  arch: [python-evdev]
```

---

How do you know that this is Python 3?

---

The python version matches the system. Arch switch to python3 a while ago.